tests/testthat/test-target_market_share.R

```diff
@@ -676,3 +676,19 @@ test_that("for one company with multiple loans of different size, unweighted
 
   expect_equal(projected$production, fake_ald()$production)
 })
+
+test_that("with bad column errors with informative message (#267)", {
```

Comment:
Can we use "column" (prefer) or "names" not both?

Reply:
Sure, although we have an old set of tests with the class `missing_names`, so likely we should fix this as well. Not sure if that should happen in this PR

Reply:
I'm happy to get that refactoring here. To me, fighting rot is important and doing it here may reduce the friction.